### PR TITLE
Rukpak dockerfile_path

### PR DESCRIPTION
**Pull request by dtfranz**
Change Dockerfile path for rukpak build to new location.

### CODE REVIEW

**Comment by tmshort** [comment]
A bit confused... I'm not seeing the `openshift` path in either rukpak repo?

**Comment by dtfranz** [comment]
/hold
Until this PR is merged: https://github.com/openshift/operator-framework-rukpak/pull/23

**Review by grokspawn — APPROVED** [review]

**Pull request merged**
